build/package-generators/curlconverter.js

```diff
@@ -7,6 +7,7 @@ const path = require('path')
 const buildDir = execEnv.buildDir
 const packageJsonPath = path.join(buildDir, 'package.json')
 const packageLockPath = path.join(buildDir, 'package-lock.json')
+const generatorsDir = path.join(buildDir, "src/generators");
```

Comment:
To make it more general, I would like to pass each path component as a separate argument.

```suggestion
const generatorsDir = path.join(buildDir, "src", "generators");
```